### PR TITLE
[fix](memory) Fix Allocator release memory to correct MemTracker after TLS attach task ends

### PR DESCRIPTION
**Pull request by xinyiZzz**
Allocator save TLS MemTracker during first alloc, which is used to release memory after TLS attach task ends.

### CODE REVIEW

**Comment by xinyiZzz** [comment]
run buildall

**Comment by xinyiZzz** [comment]
run buildall

**Comment by xinyiZzz** [comment]
run buildall

**Comment by xinyiZzz** [comment]
run buildall

**Comment by xinyiZzz** [comment]
run buildall

**Comment by xinyiZzz** [comment]
run buildall

**Comment by xinyiZzz** [comment]
run buildall

File: be/src/vec/common/allocator.h [hunk 1]
```diff
@@ -222,13 +226,20 @@ class RecordSizeMemoryAllocator {
 template <bool clear_memory_, bool mmap_populate, bool use_mmap, typename MemoryAllocator>
 class Allocator {
 public:
+    Allocator();
+    Allocator(const Allocator& that) { tracker = that.tracker; }
```
Comment (yiguolei) [inline be/src/vec/common/allocator.h:230]: 这个改写如果没有作用的话，先删了。

**Comment by xinyiZzz** [comment]
run buildall

File: be/src/vec/common/allocator.h (hunk 1 above)
Comment (xinyiZzz) [reply be/src/vec/common/allocator.h]: fixed

**Comment by xinyiZzz** [comment]
run buildall

**Comment by xinyiZzz** [comment]
run buildall

**Comment by xinyiZzz** [comment]
run buildall

**Pull request closed without merging**